src/bootstrap/doc.rs

```diff
@@ -70,6 +70,35 @@ book!(
     RustdocBook, "src/doc/rustdoc", "rustdoc";
 );
 
+fn open(builder: &Builder<'_>, path: impl AsRef<Path>) {
+    if builder.config.dry_run || !builder.config.cmd.open() {
+        return;
+    }
+
+    let path = path.as_ref();
+    builder.info(&format!("Opening doc {}", path.to_string_lossy()));
+
+    // ignore error
+    let _ = opener::open(path);
```

Comment:
I would have expected us to print the error here - at least on higher levels of verbosity, is there a reason we're ignoring it? I imagine it'd be useful for people who try to find out why open doesn't work for them.

Reply:
Done. On my machine the error looks like:

```console
    Finished release [optimized] target(s) in 0.29s
Opening doc /git/rust/build/x86_64-unknown-linux-gnu/doc/std/index.html
command 'xdg-open (internal)' did not execute successfully; exit code: 4
command stderr:
gio: file:///git/rust/build/x86_64-unknown-linux-gnu/doc/std/index.html: Error when getting information for file “/git/rust/build/x86_64-unknown-linux-gnu/doc/std/index.html”: No such file or directory

Build completed successfully in 0:00:08
```